utils/utils.test.ts

```diff
@@ -50,10 +50,12 @@ const mockGameCurrentWeek = {
 
 const mockWeeklyPicksData = {
```

Comment:
@chris-nowicki This test was updated because it had to be commented out from pr #477 so I had to update the data structure of this. Please review the test and let me know what your thought is.

Reply:
understood :)